html/pfappserver/lib/pfappserver/Form/Config/Authentication/Rule.pm

```diff
@@ -145,6 +145,22 @@ has_field "${Conditions::TIME}_value" =>
    wrapper => 0,
    element_class => ['span5'],
   );
+has_field "${Conditions::TIME_PERIOD}_operator" =>
+  (
+   type => 'Select',
+   do_label => 0,
+   wrapper => 0,
+   localize_labels => 1,
+   options_method => \&operators,
+   element_class => ['span3'],
+  );
+has_field "${Conditions::TIME_PERIOD}_value" =>
+  (
+   type => 'Text',
```

Comment:
We should change this to a *select* field and predefine a few common time periods. Otherwise the user must know the format to define a time period.